src/streaming.ts

```diff
@@ -212,14 +228,18 @@ export class StreamProcessor {
     return this.readyState === StreamProcessor.CONNECTED;
   }
 
-  stop(): void {
-    this.log.info('Stopping StreamProcessor');
-    this.eventBus.emit(StreamEvent.DISCONNECTED);
-  }
```

Comment:
Looks like this was a public function. Is this change potentially breaking and require a `2.0.0`?